packages/next/build/webpack-config.ts

```diff
@@ -71,7 +71,7 @@ const babelIncludeRegexes: RegExp[] = [
   /next[\\/]dist[\\/](esm[\\/])?client/,
   /next[\\/]dist[\\/](esm[\\/])?pages/,
   /[\\/](strip-ansi|ansi-regex)[\\/]/,
-  /styled-jsx[\\/]/,
+  /next[\\/]dist[\\/]styled-jsx[\\/]/,
```

Comment:
We don't have assets in `next/dist` but only types. maybe you can use `/[\\/]/styled-jsx[\\/]` to indicate it's `styled-jsx` folder?

Reply:
And it can then be merged with the `strip-ansi` and `ansi-regex` above.